sklearn/linear_model/tests/test_omp.py

```diff
@@ -30,6 +30,34 @@
 # and y (n_samples, 3)
 
 
+# FIXME: 'normalize' to be removed in 1.2
```

Comment:
Or modified after 1.2 to just check for the usual FutureWarning before removal of the parameter?